auth-web/tests/unit/components/EntityManagement.spec.ts

```diff
@@ -84,7 +86,7 @@ describe('Entity Management Component', () => {
   })
 
   it('EntityManagement is a Vue instance', () => {
-    expect(wrapper.isVueInstance()).toBeTruthy()
+    expect(wrapper.vm).toBeTruthy()
```

Comment:
isVueInstance() is [depreciated](https://v1.test-utils.vuejs.org/api/wrapper/isvueinstance.html)